store/sqlstore/upgrade.go

```diff
@@ -575,8 +576,14 @@ func UpgradeDatabaseToVersion59(sqlStore SqlStore) {
 }
 
 func UpgradeDatabaseToVersion510(sqlStore SqlStore) {
-	// if shouldPerformUpgrade(sqlStore, VERSION_5_9_0, VERSION_5_10_0) {
-
-	// 	saveSchemaVersion(sqlStore, VERSION_5_10_0)
-	// }
+	if shouldPerformUpgrade(sqlStore, VERSION_5_9_0, VERSION_5_10_0) {
```

Comment:
Ok, you undestand us wrong, that should be the way to reproduce the error, but shouldn't be in the code. And the solution to the error, probably should be in the main "UpgradeDatabase" function. Ideally, I picture it as a `SET statement_timeout TO 0` at the beginning, and restore the original value at the end (probably you need to get the value from the database, do the set, and then a defer with a function that restores the value).

Reply:
@jespino is correct. @joshuabezaleel, feel free to ping @jespino or myself on https://community.mattermost.com for further clarification. Happy to help!